integration-cli/docker_cli_nat_test.go

```diff
@@ -60,18 +55,14 @@ func (s *DockerSuite) TestNetworkNat(c *check.C) {
 	startServerContainer(c, msg, 8080)
 	endpoint := getExternalAddress(c)
 	conn, err := net.Dial("tcp", fmt.Sprintf("%s:%d", endpoint.String(), 8080))
-	if err != nil {
-		c.Fatalf("Failed to connect to container (%v)", err)
-	}
+	c.Assert(err, check.IsNil)
+
 	data, err := ioutil.ReadAll(conn)
 	conn.Close()
-	if err != nil {
-		c.Fatal(err)
-	}
+	c.Assert(err, check.IsNil)
+
 	final := strings.TrimRight(string(data), "\n")
-	if final != msg {
-		c.Fatalf("Expected message %q but received %q", msg, final)
-	}
+	c.Assert(final, check.Equals, msg, check.Commentf("Expected message %q but received %q", msg, final))
```

Comment:
The error comment is nothing special so I think c.Assert(final, checker.Equals, msg) is OK.